### ISSUE TITLE
🐛 useInlineScriptId: false positive when <Script> opening tag   spans multiple lines

### ISSUE THREAD

**Issue by wadehammes** [biomejs/biome#10733]
### Environment information

- **Biome version:** 2.5.0
- **Rule:** `lint/correctness/useInlineScriptId` (Next.js domain)
- **Domain:** `next` (auto-detected via `next` in dependencies; Next.js 16.2.9)
- **Platform:** macOS (darwin)

### What happened?

<img width="800" height="297" alt="Image" src="https://github.com/user-attachments/assets/146d04cf-5e07-403b-a2f4-f50e34802a78" />

`useInlineScriptId` reports `next/script components have inline content or dangerouslySetInnerHTML without id attribute` on a `<Script>` element **that does have an `id` attribute**, whenever the opening tag is formatted across **multiple lines** (one attribute per line).

When the exact same element — same attributes, same `id`, same inline children — is written on a **single line**, the rule correctly passes. So the diagnostic is driven by the formatting/line-spanning of the opening tag, not by the presence or absence of `id`.

This is easy to hit in practice because Biome's own formatter wraps a `<Script>` opening tag onto multiple lines as soon as it has 3+ attributes or exceeds the line width — at which point this rule starts firing on correct code.

## Steps to reproduce

> **Note:** this cannot be reproduced in the online Biome playground. The rule lives in the `next` domain, which is only activated when the project manifestdeclares `next` as a dependency (or when `linter.domains.next` is set explicitly). The playground has no manifest and no domain toggle, so the rule never runs there. Use a local project with the config below instead.

Minimal standalone reproduction — `biome.json` that forces the domain on:

```json
{
  "linter": {
    "enabled": true,
    "domains": { "next": "all" },
    "rules": { "recommended": true }
  }
}
```

**Fires the false positive** — opening tag spans multiple lines, `id` present:

```tsx
import Script from "next/script";

export default function Page() {
  return (
    <Script
      id="my-script"
      strategy="lazyOnload"
    >
      {`console.log("hello");`}
    </Script>
  );
}
```

```
lint/correctness/useInlineScriptId ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
  × next/script components have inline content or `dangerouslySetInnerHTML` without id attribute.
```

**Passes (correct)** — identical element on a single line:

```tsx
<Script id="my-script" strategy="lazyOnload">
  {`console.log("hello");`}
</Script>
```

Holding the `id` attribute constant and present in every case:

| Opening tag | Attributes | Result |
|-------------|-----------|--------|
| single line | `id`, `strategy` (2) | ✅ passes |
| single line | `id`, `data-a`, `data-b` (3) | ✅ passes |
| multi-line  | `id`, `strategy` (2) | ❌ false positive |
| multi-line  | `id`, `strategy`, `type` (3) | ❌ false positive |

The only variable that changes the outcome is whether the opening tag is on one line or wrapped across several. Attribute count and which attributes are present (e.g. `type`) do not matter on their own.

### Expected result

The rule should detect the `id` attribute regardless of how the opening tag is formatted. A multi-line opening tag with `id="..."` present should not be reported.

### Actual behavior

A `<Script>` with inline content and an `id` attribute is reported as missing `id` whenever its opening tag spans multiple lines.

### Suspected cause

In `crates/biome_js_analyze/src/lint/correctness/use_inline_script_id.rs`, the rule collects attribute names into a set and then checks `attribute_names.contains("id")`. The collection reads each attribute name via `name_value_token()` / `token_text_trimmed()`. The attribute-name matching appears to break when attributes carry newline/indentation trivia (i.e. when the tag is formatted multi-line), so `"id"` is not matched even though it is present. There is no special handling for `type`; the observable trigger is purely the multi-line formatting of the opening tag.

(Possibly related to other multi-line-JSX handling issues such as #9469, though that is a different rule.)

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by Netail** [comment]
Duplicate of #10651. Has been fixed on main and will be released in the upcoming patch :)
